styles/index.scss

```diff
@@ -1,6 +1,25 @@
+html {
+  box-sizing: border-box;
+}
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+
```

Comment:
We should consider moving these out of this file and into some sort of reset partial.
`index.scss` as we have it now should not compile to anything and just contain scss variables, mixins, functions.